base/default.StorageClass.yaml

```diff
@@ -0,0 +1,14 @@
+kind: StorageClass
```

Comment:
I don't think this should be included here because it is invalid. I think it would be sufficient to just inline this fragment into the README itself.